packages/react-components/react-data-grid-react-window/src/components/DataGrid/DataGrid.tsx

```diff
@@ -4,12 +4,24 @@ import {
   renderDataGrid_unstable,
   useDataGridStyles_unstable,
   useDataGridContextValues_unstable,
+  TableSelectionCellWidth,
 } from '@fluentui/react-table';
 import type { DataGridProps } from '@fluentui/react-table';
-import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { ForwardRefComponent, useScrollbarWidth } from '@fluentui/react-utilities';
+import { useFluent } from '@fluentui/react-components';
 
 export const DataGrid: ForwardRefComponent<DataGridProps> = React.forwardRef((props, ref) => {
-  const state = useDataGrid_unstable(props, ref);
+  const { targetDocument } = useFluent();
+  const scrollbarWidth = useScrollbarWidth({ targetDocument });
```

Comment:
Since you're going to be using scrollbar width internally now, could you also try to get rid of this usage? https://github.com/microsoft/fluentui/blob/8ec648d4060906a32bad46a16e0eae13a7aa5959/packages/react-components/react-table/stories/DataGrid/Virtualization.stories.tsx#L159

Reply:
yup, make sense:)

Reply:
follow up ticket https://github.com/microsoft/fluentui/issues/28027